doc/example/model_selection.ipynb

```diff
@@ -0,0 +1,828 @@
+      "text/plain": [
+       "<Figure size 864x864 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "initial_model_label = {\n",
+    "    initial_model.get_hash(): initial_model.model_id\n",
+    "}\n",
+    "\n",
+    "pvs.plot_history_digraph(\n",
+    "    problem=pypesto_select_problem_2,\n",
+    "    labels={\n",
+    "        **get_digraph_labels(pypesto_select_problem_2.history.values(), criterion=Criterion.AIC),\n",
+    "        **initial_model_label,\n",
+    "    },\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Additional Options\n",
+    "There exist additional options that can be used to further customise selection algorithms.\n",
+    "### Select First Improvement\n",
+    "At each selection step, as soon as a model that improves on the previous model is encountered (by the specified criterion), it is selected and immediately used as the previous model in the next iteration of the selection. This is unlike the default behaviour, where all test models at each iteration are optimized, and the best of these is selected.\n",
+    "### Use Previous Maximum Likelihood Estimate as Startpoint\n",
+    "The maximum likelihood estimate parameters from the previous model is used as one of the startpoints in the multistart optimization of the test models. The default behaviour is that all startpoints are automatically generated by pyPESTO.\n",
+    "### Minimize Options\n",
+    "Optimization can be customised with a dictionary that specifies values for the corresponding keyword arguments of [minimize](https://github.com/ICB-DCM/pyPESTO/blob/master/pypesto/optimize/optimize.py).\n",
```

Comment:
[minimize] could be linked to docs via RST cell too here. but github is fine too I guess.